specification/paths/Carriers-carrier_id-CarrierContracts.json

```diff
@@ -3,6 +3,16 @@
     "tags": [
       "Carriers"
     ],
+    "security": [{
+      "OAuth2": [
+        "shipments.show",
+        "shipments.manage",
+        "brokers.show",
+        "brokers.manage",
+        "brokers.manage",
```

Comment:
3x `brokers.manage`.

Missing `organizations.manage` and `organizations.show`.